### ISSUE TITLE
For Learning Objects add cognitive dimension to \symref and friends

### ISSUE THREAD

**Issue by kohlhase** [slatex/sTeX#394]
In VoLL-KI learning objects, symbol references might want to carry other cognitive dimensions than just remember (as in the domain model). Currently we would have to write something like
```
\objective{create}{search problem}
\prerequisite{understand}{csp}
Make a \sn{search problem} from the following \sn{csp}...
```
I am OK with stating the learning objective in this way, but I would like to get rid of hte prerequisite by just writing `\sn[u]{csp}`
by abbreviating the cognitive dimension by their first letter (except for `analyze` which gets an `n` and `remember` which is the default). 
I think this would make our work much easier.

**Comment by Jazzpirate** [comment]
I disagree completely (except for the abbreviations-part - I disagree there too, but might be more easily convinced there. But I hate it when the result is that sTeX becomes entirely unreadable to the "uninitiated" and requires memorizing a bunch of abbreviations to have the faintest idea of what's going on. Replacing all \symnames/\symrefs by `\sn` and `\sr` was a giant mistake - it's one thing to use it as a power user, another to actively make sure that all content only consists of abbreviations.)

1. I don't think it's a good idea to crowbar vollki-specific annotations into default ubiquitous sTeX-macros, both already take key-value-pairs in `[]`, I don't think changing the syntax globally is warranted, and patching the macros locally is even worse,
2. it conceptually makes no sense to attach a cognitive dimension on a symbol reference, because it's just a symbol reference. The dimension has no meaning there.

It's not at all clear that a reference entails a precondition - the idea that "references entail remember-prerequisites" is a heuristic that I suspect will turn out to be flawed in some edge cases. And the prerequisites we do annotate, I would like to be explicit especially to distinguish them from the "guessed" ones.
